rollup build is failing with not finding input (v0.10.0-alpha.2)

## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
After upgrading to the latest alpha release of Greenwood, getting this error
```
Error: You must supply options.input to rollup
    at Graph.generateModuleGraph (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/rollup/dist/shared/rollup.js:18908:19)
    at async Graph.build (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/rollup/dist/shared/rollup.js:18872:9)
    at async rollupInternal (/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/rollup/dist/shared/rollup.js:19947:9)
    at async /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/lifecycles/bundle.js:12:24
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! greenwood-getting-started@1.0.0 build: `greenwood build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the greenwood-getting-started@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/owenbuckley/.npm/_logs/2020-12-30T21_18_46_431Z-debug.log
```


## Details
Some debugging details the following:
```js
module.exports = getRollupConfig = async (compilation) => {
  
  const { scratchDir, outputDir } = compilation.context;

  console.debug('getRollupConfig scratchDir?????????????', `${scratchDir}/**/*.html`);
  console.debug('getRollupConfig outputDir?????????????', `${outputDir}/**/*.html`);

  return [{
    // TODO Avoid .greenwood/ directory, do everything in public/?
    input: `${scratchDir}/**/*.html`,
```

```sh
getRollupConfig scratchDir????????????? /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/.greenwood//**/*.html
getRollupConfig outputDir????????????? /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/public/**/*.html
```

Looks like there are two `//` for `scratchDir`?  When removing it, everything is fine again.


This seems to be an issue with **npm** based projects, since I had this issue in Getting Started, but not www.thegreenhouse.io

--- a/packages/cli/src/config/rollup.config.js
+++ b/packages/cli/src/config/rollup.config.js
@@ -224,7 +224,7 @@ module.exports = getRollupConfig = async (compilation) => {
 
   return [{
     // TODO Avoid .greenwood/ directory, do everything in public/?
-    input: `${scratchDir}/**/*.html`,
+    input: `${scratchDir}**/*.html`,
     // preserveEntrySignatures: false,
     output: { 
       dir: outputDir,

